Add pom.xml and task from 4 sprint

Lunatik06 opened the pull request:
Добавление pom

Commit:
Add pom.xml

Commit:
Add task from 4 sprint

nikolay-977 commented on src/test/java/pages/MainPage.java:
```diff
@@ -0,0 +1,95 @@
+    }
+
+    // получить тело 2-го пункта из блока "Сколько это стоит? И как оплатить?"
+    private static final By bodyQuestion = By.xpath(".//div[@id='accordion__panel-1']/p");
```
Необходимо исправить: В начале класса следует описывать переменные, а затем конструкторы и методы. Также можно установить курсор на корневую директорию и нажать сочетание клавиш Ctrl+Alt+L (Option+Command+L для Mac), затем выбрать checkbox rearrange entries и нажать ok.

Lunatik06 replied:
Поправил в коммите https://github.com/Lunatik06/Sprint_4/pull/1/commits/3288addd7f7ac404b50012e93747e59d9c5ca0d5

nikolay-977 commented on src/test/java/pages/OrderPage.java:
```diff
@@ -0,0 +1,131 @@
+        driver.findElement(surnameInput).sendKeys(surname);
+        return this;
+    }
+
```
Можно улучшить: лишних пропусков строк следует избегать. Для зрительного разделения кода достаточно использовать один пропуск строки. Пропуски строк должны быть однообразными.

Lunatik06 replied:
Поправил в коммите https://github.com/Lunatik06/Sprint_4/pull/1/commits/3288addd7f7ac404b50012e93747e59d9c5ca0d5

nikolay-977 commented on src/test/java/pages/MainPage.java:
```diff
@@ -0,0 +1,95 @@
+
+    // получить тело 2-го пункта из блока "Сколько это стоит? И как оплатить?"
+    private static final By bodyQuestion = By.xpath(".//div[@id='accordion__panel-1']/p");
+    public MainPage checkBodyBlockQuestion() {
```
Можно улучшить: для расстановки пробелов и табуляции в соответствии с настройками Code Style в Idea используй сочетание клавиш Ctrl+Alt+L. Также можно установить курсор на корневую директорию и нажать клавиш Ctrl+Alt+L, отметить чек-боксы: cleanup code, а также optimize imports(для оптимизации импортов) и rearrange entries(для изменения порядка переменных и методов в соответствии с best practics)

Lunatik06 replied:
поправил в коммите https://github.com/Lunatik06/Sprint_4/pull/1/commits/3288addd7f7ac404b50012e93747e59d9c5ca0d5

nikolay-977 reviewed (commented):
Можно улучшить: хорошей практикой является добавление README.md с описанием проекта, которое включает название, технологии в проекте (например, Java 11, JUnit 4.13.2, maven 3.9.0 и т.д.), настройка (если необходима) и как запускать (`mvn clean test` - кавычки позволяют выполнять запуск из файла README.md). Как оформлять README.md можно ознакомиться в статье https://blog.jetbrains.com/idea/2022/08/markdown-support-improvements/

Commit:
- переделан readme, добавлено описание, технологии, команда для запуска  тестов
- применена автоматическая операции, cleanup code, optimize imports, rearrange entries
- убраны неиспользуемые куски, которые остались от работы на вебинаре и не нужны в проекте

Lunatik06 commented:
Добавил новый коммит, в котором добавил readme и внёс другие правки по сообщениям выше https://github.com/Lunatik06/Sprint_4/pull/1/commits/3288addd7f7ac404b50012e93747e59d9c5ca0d5
